app/routes/index.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
```

Comment:
We don't need this anymore since the logic is in the reminders route - this file should be used to redirect to `'/reminders'`